larq_compute_engine/core/maxpool.h

```diff
@@ -0,0 +1,91 @@
+#ifndef LARQ_COMPUTE_ENGINE_CORE_MAXPOOL_H_
+#define LARQ_COMPUTE_ENGINE_CORE_MAXPOOL_H_
+
+#include "tensorflow/lite/kernels/internal/common.h"
+#include "tensorflow/lite/kernels/internal/types.h"
+#include "tensorflow/lite/kernels/padding.h"
+
+using namespace tflite;
+
+namespace compute_engine {
+namespace core {
+
+struct BMaxPool2DParams {
```

Comment:
I'm not sure what the best-practice is here.
In our case we will fill all values in the `Init` step. I don't think there are simple default values that make sense, like you could set `filter_height=2` because its very common, but I think that'll only cause unforeseen bugs.

Reply:
I would initialize with zero as much as possible as we do for bconv2d op [here](https://github.com/larq/compute-engine/blob/63eda98f7d5c9e5b525a5445fab2acf2d03ce4b1/larq_compute_engine/tflite/kernels/bconv2d_params.h#L15)